reana_commons/consumer.py

```diff
@@ -21,51 +21,50 @@
 # submit itself to any jurisdiction.
 """REANA-Commons module to manage AMQP consuming on REANA."""
 
-import pika
+from kombu import Connection, Exchange, Queue
+from kombu.mixins import ConsumerMixin
 
-from .config import BROKER_PASS, BROKER_PORT, BROKER_URL, BROKER_USER
+from reana_commons.config import (BROKER, MQ_DEFAULT_EXCHANGE,
+                                  MQ_DEFAULT_QUEUE, MQ_DEFAULT_ROUTING_KEY,
+                                  MQ_DEFAULT_SERIALIZER)
 
 
-class Consumer:
-    """Base MQ Consumer."""
+class REANABaseConsumer(ConsumerMixin):
```

Comment:
This class is in the same category as the `BaseAPIClient`, so the names could have the same system. Either we add `REANA...` to the BaseAPIClient or we remove it here. I would say we can remove it and just call it `BaseConsumer` for brevity.